test/gtest/ucp/test_ucp_tag_offload.cc

```diff
@@ -539,6 +539,30 @@ UCS_TEST_P(test_ucp_tag_offload_cuda, sw_rndv_to_cuda_mem, "TM_SW_RNDV=y")
 UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_tag_offload_cuda, rc_dc_cuda,
                               "dc_x,rc_x,cuda_copy")
 
+class test_ucp_tag_offload_status : public test_ucp_tag {
+public:
+    test_ucp_tag_offload_status() {
+        m_env.push_back(new ucs::scoped_setenv("UCX_RC_TM_ENABLE", "y"));
+    }
+
+    static ucp_params_t get_ctx_params() {
+        ucp_params_t params = ucp_test::get_ctx_params();
+        params.features     = std::numeric_limits<uint64_t>::max() &
```

Comment:
maybe use explicit feature like RMA and comment and TAG is not passed?

Reply:
my intent was to test that only TAG feature enables tag offload. This check would work even if new features was added.
change to RMA?

Reply:
i think it would be better, seems weird we can pass even bits which are not defined, at some point i would even fail ucp_init() if unsupported features are passed